Dockerfile

```diff
@@ -2,10 +2,13 @@ FROM centos:centos7
 
 EXPOSE 8080
 
+RUN curl --retry 999 --retry-max-time 0 -sSL https://github.com/openshift/origin/releases/download/v1.2.0/openshift-origin-client-tools-v1.2.0-2e62fab-linux-64bit.tar.gz | tar xzv && \
+  mv openshift-origin-*/* /usr/bin/
```

Comment:
> Does this seem like a reasonable way to install the client?

I think its best to install via yum, Andrea will thank us. The package is called openshift-cli, IIRC.